.actor/actor.json

```diff
@@ -0,0 +1,50 @@
+{
+    "actorSpecification": 1,
+    "name": "sandbox",
+    "title": "Isolated Sandbox Actor",
+    "description": "Isolated sandbox Actor that executes code and operations in a containerized environment for AI coding agents",
+    "version": "0.0",
+    "buildTag": "latest",
+    "usesStandbyMode": false,
+    "meta": {
+        "templateId": "ts-standby",
+        "generatedBy": "claude-haiku-4.5"
+    },
+    "input": {
+        "title": "Actor input schema",
+        "description": "Isolated sandbox for running AI coding operations in a containerized environment. 🚀\n\n## How to use\n* **Connect with MCP client:** Use a Model Context Protocol (MCP) client to interact with this sandbox. See https://modelcontextprotocol.io/clients\n* **Use Apify SDK:** Connect programmatically using the Apify SDK for direct integration\n* **REST API:** Access the sandbox directly via REST API endpoints\n\n## Configuration\n* **Memory & timeout:** Configure run options below to set memory allocation and execution timeout\n* **Check logs:** Open the Actor run log console to view connection details and operation output",
+        "type": "object",
+        "schemaVersion": 1,
+        "properties": {
+            "nodeLibraries": {
+                "title": "Node.js Libraries",
+                "type": "array",
+                "description": "Array of strings: Enter Node.js/npm libraries to install for JavaScript and TypeScript code execution (/sandbox/js-ts). Example: [\"zod\", \"axios\"]",
+                "items": {
+                    "type": "string"
+                },
+                "prefill": [],
+                "editor": "json"
+            },
+            "pythonLibraries": {
+                "title": "Python Libraries",
+                "type": "array",
+                "description": "Array of strings: Enter Python libraries (pip packages) to install for Python code execution (/sandbox/py). Example: [\"requests\", \"pandas\"]",
```

Comment:
Similar as above - is the JSON array the most convenient format for this? How about having a text input where you could write anything that works inside requirements.txt? It's basically one package per file, and supports pinning versions (`requests==2.31.0`). It could be called `pythonRequirementsTxt`